backend/games/word_guesser/WordGuesser_test.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+
+from backend.games.word_guesser.WordGuesser import WordGuesser
+
+SOLUTIONS: str
+
+solutions_file = open("backend/games/word_guesser/WordGuesser_Solutions.txt")
+guesses_file = open("backend/games/word_guesser/WordGuesser_ValidGuesses.txt")
```

Comment:
How did you get these lists? We should attribute the source of the list, at least in the source code.

Reply:
Pulled both lists from a public github repo